src/components/NavBar/NavBar.css

```diff
@@ -0,0 +1,36 @@
+.nav-bar {
+    display: flex;
+    justify-content: flex-end;
+    width: 100vw;
+    margin: 0 auto;
+    background-color: #E5E5E5;
```

Comment:
In the mock the navbar has a white background, when I render it on my computer it is grey. Can we fix that.

Reply:
I actually got that background color from the mock.  While it does look white, it says the background color is #E5E5E5

Reply:
You sir are right the mock is calling for that color. Will have to touch base with the designer, but for now lets keep what the mock says.

Reply:
You sir are right the mock is calling for that color. Will have to touch base with the designer, but for now lets keep what the mock says.